### PR TITLE
Analytics zoo inference scala

### PR DESCRIPTION
**Pull request by yangw1234**

### CODE REVIEW

File: scala-inference/pom.xml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    <artifactId>mem-inference</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <java.version>1.8</java.version>
+        <scala.major.version>2.11</scala.major.version>
+        <scala.version>2.11.8</scala.version>
+        <scalatest.version>2.2.4</scalatest.version>
+        <spark.major.version>2.1</spark.major.version>
+        <spark.version>2.1.1</spark.version>
```
Comment (Dooyoung-Hwang) [inline scala-inference/pom.xml]: We have been using spark 2.3.3 for testing. Could you change this accordingly?

File: scala-inference/pom.xml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        <dependency>
+            <groupId>com.intel.analytics.zoo</groupId>
+            <artifactId>analytics-zoo-bigdl_0.8.0-spark_${spark.version}</artifactId>
+            <version>0.5.1</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jetbrains.bio</groupId>
+            <artifactId>npy</artifactId>
+            <version>0.3.3</version>
```
Comment (Dooyoung-Hwang) [inline scala-inference/pom.xml]: This artifact cause build-failure with default maven setting. Could you add a description of spring lib release repository to this file?
```
<repositories>
    <repository>
        <id>spring-lib-release</id>
        <name>Spring Lib Release Repository</name>
        <url>https://repo.spring.io/libs-release</url>
    </repository>
</repositories>
```

File: scala-inference/pom.xml (hunk 1 above)
Comment (yangw1234) [reply scala-inference/pom.xml]: sure

File: scala-inference/pom.xml (hunk 2 above)
Comment (yangw1234) [reply scala-inference/pom.xml]: sure

File: scala-inference/pom.xml (hunk 1 above)
Comment (yangw1234) [reply scala-inference/pom.xml]: analytics-zoo only released the spark 2.3.1 version. I have tested on spark 2.3.3. it worked.

File: scala-inference/pom.xml (hunk 2 above)
Comment (yangw1234) [reply scala-inference/pom.xml]: added

**Comment by Dooyoung-Hwang** [comment]
Is there any more update for this PR? Do we merge to analytics-zoo-inference branch?

**Comment by yangw1234** [comment]
@Dooyoung-Hwang currently no update to the pr

**Comment by yangw1234** [comment]
I'll merge it to the analytics-zoo-inference branch

**Pull request merged**